media-libs/libnsbmp/libnsbmp-0.1.2.recipe

```diff
@@ -1,21 +1,30 @@
-DESCRIPTION="Libnsbmp is a decoding library for BMP and ICO image file formats"
 SUMMARY="A decoding library for BMP and ICO images"
+DESCRIPTION="Libnsbmp is a decoding library for BMP and ICO image file formats"
 HOMEPAGE="http://www.netsurf-browser.org/projects/libnsbmp/"
 COPYRIGHT="2006 Richard Wilson, 2008 - 2013 Sean Fox"
 LICENSE="MIT"
+REVISION="1"
 SOURCE_URI="http://download.netsurf-browser.org/libs/releases/libnsbmp-$portVersion-src.tar.gz"
 CHECKSUM_SHA256="969ba1c4f778b6ecee5fd834a6206c97b49885673389260fef1043dfca2968b1"
-REVISION="1"
-ARCHITECTURES="x86_gcc2 x86 x86_64"
-
 PATCHES="libnsbmp-0.1.1.patchset"
 
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
 PROVIDES="
 	libnsbmp = $portVersion
 	"
 REQUIRES="
 	haiku
 	"
+	
+PROVIDES_devel="
+	libnsbmp_devel = $portVersion
+	devel:libnsbmp = $portVersion
+	"
+REQUIRES_devel="
+	libnsbmp$secondaryArchSuffix == $portVersion base
```

Comment:
Delete this `$secondaryArchSuffix`, there are no SECONDARY_ARCHITECTURES specified.